tf-psa-crypto/drivers/builtin/include/mbedtls/oid.h

```diff
@@ -466,10 +466,6 @@
 #define MBEDTLS_OID_ED25519                 MBEDTLS_OID_THAWTE "\x70" /**< id-Ed25519   OBJECT IDENTIFIER ::= { 1 3 101 112 } */
 #define MBEDTLS_OID_ED448                   MBEDTLS_OID_THAWTE "\x71" /**< id-Ed448     OBJECT IDENTIFIER ::= { 1 3 101 113 } */
 
-#ifdef __cplusplus
```

Comment:
There seem to be some artifacts from the rebase here (and also in lines 560, 606, 723). I help to address these and review after that.

Reply:
I have corrected the rebase, this error shouldn't be propogated further now.